pyplanet/apps/contrib/funcmd/__init__.py

```diff
@@ -62,6 +115,22 @@ async def command_afk(self, player, data, **kwargs):
 			self.instance.chat('$fff {}$z$s$fff is now away from keyboard.'.format(player.nickname))
 		)
 
+	async def command_muffin(self, player, data, **kwargs):
+		if self.instance.game.game == 'tmnext':
+			dest_player = [p for p in self.instance.player_manager.online if p.nickname == data.login]
```

Comment:
I would suggest making the nickname comparison case insensitive. The scoreboard in TM2020 uses $t for players so the exact casing of a given nickname can be hard to know.

Reply:
Nickname of a player is based on the Ubisoft Connect login it seems:
```

[2023/08/29 22:48:46] <chat> [q2-lckjXSxai11x2CgX5ew (W1llaSmurf)] /muffin
[2023/08/29 22:48:46] <chat> [zaVkURODRPGS-SWP_pUwGQ (W1lla Relay#1)] $z$s$fff» $z$sCommand operation got invalid arguments: param 'login' is required
[2023/08/29 22:48:46] <chat> [zaVkURODRPGS-SWP_pUwGQ (W1lla Relay#1)] $z$s$fff» $z$s >> Usage: /muffin login:str
[2023/08/29 22:48:53] <chat> [q2-lckjXSxai11x2CgX5ew (W1llaSmurf)] /muffin w1llasmurf
[2023/08/29 22:48:53] <chat> [zaVkURODRPGS-SWP_pUwGQ (W1lla Relay#1)] $z$s$fff» $i$f00Unknown player!
[2023/08/29 22:49:11] <chat> [q2-lckjXSxai11x2CgX5ew (W1llaSmurf)] /muffin W1llaSmurf
[2023/08/29 22:49:11] <chat> [zaVkURODRPGS-SWP_pUwGQ (W1lla Relay#1)] $z$s$fff»» $zW1llaSmurf$z$m$f90 gave Carrot Cake to W1llaSmurf
[2023/08/29 22:49:21] <chat> [q2-lckjXSxai11x2CgX5ew (W1llaSmurf)] /muffin W1llaSmurf
[2023/08/29 22:49:21] <chat> [zaVkURODRPGS-SWP_pUwGQ (W1lla Relay#1)] $z$s$fff»» $zW1llaSmurf$z$m$f90 gave Slice of Cheesecake to W1llaSmurf
[2023/08/29 22:49:29] <chat> [q2-lckjXSxai11x2CgX5ew (W1llaSmurf)] /muffin W1llaSmurf
[2023/08/29 22:49:29] <chat> [zaVkURODRPGS-SWP_pUwGQ (W1lla Relay#1)] $z$s$fff»» $zW1llaSmurf$z$m$f90 gave Cinnamon Streusel Muffin to W1llaSmurf
```